src/backend/src/prisma/seed.ts

```diff
@@ -237,13 +237,43 @@ const performSeed: () => Promise<void> = async () => {
     changeRequest1.proposedSolutions[0].id
   );
 
+  /** Gets the current content of the .env file */
+  const currentEnv = require('dotenv').config().parsed;
+
+  currentEnv.DEV_ORGANIZATION_ID = organizationId;
+
+  /** Write the new .env file with the organization ID */
+  let stringifiedEnv = '';
+  Object.keys(currentEnv).forEach((key) => {
+    stringifiedEnv += `${key}=${currentEnv[key]}\n`;
+  });
+  writeFileSync('.env', stringifiedEnv);
+
   /**
    * TEAMS
    */
   /** Creating Team Types */
-  const teamType1 = await TeamsService.createTeamType(batman, 'Mechanical', 'YouTubeIcon', organizationId);
-  const teamType2 = await TeamsService.createTeamType(thomasEmrax, 'Software', 'InstagramIcon', organizationId);
-  const teamType3 = await TeamsService.createTeamType(cyborg, 'Electrical', 'SettingsIcon', organizationId);
+  const teamType1 = await TeamsService.createTeamType(
+    batman,
+    'Mechanical',
+    'YouTubeIcon',
+    organizationId,
+    currentEnv.MECHANICAL_CALENDAR
```

Comment:
These aren't in the env file and we probably dont need to have them there, so you can make the types without supplying calendar id